docs/components/overview.md

```diff
@@ -10,7 +10,7 @@ This section contains product manual content for each component in Camunda Cloud
 - [Concepts](concepts/what-is-camunda-cloud.md) - Conceptual documentation on a variety of Camunda Cloud topics.
 - [Clients](../apis-clients/overview.md) - Supported clients for working with Camunda Cloud, including information on building your own.
 - [Cloud Console](cloud-console/introduction.md) - More information on working with Cloud Console.
-- [Modeler](modeler/overview.md) - Documentation on Camunda's modeling tools, including Cloud Modeler and Camunda Modeler.
+- [Modeler](modeler/what-is-camunda-modeler.md) - Documentation on Camunda's modeling tools, including Cloud Modeler and Camunda Modeler.
```

Comment:
Consider renamining to `about`?

Reply:
Do you prefer `about`? How do you feel about `using-camunda-modeler` or `working-with-camunda-modeler` or `applying-camunda-modeler`?

Reply:
Given that it is the overview page, I think something generic like `about` is best. Also considering that you'll not be able to go around our `modeler(s)` anyway, as you do anything Camunda.